exasol/toolbox/tools/security.py

```diff
@@ -15,8 +15,9 @@
     Iterable,
     Tuple,
 )
-
+from noxconfig import PROJECT_CONFIG
```

Comment:
usually we do not want to import from nox config except withint the noxfile.
(Just remove the default)